src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -131,23 +131,13 @@ def test_run_conversion_with_backend(self, setup_interface, tmp_path, backend):
                 io.read()
 
     @pytest.mark.parametrize("backend", ["hdf5", "zarr"])
-    def test_run_conversion_with_backend_configuration(self, setup_interface, tmp_path, backend):
+    def test_create_backend_configuration(self, setup_interface, tmp_path, backend):
```

Comment:
Those tests were not doing anything. See the bug described here:

https://github.com/catalystneuro/neuroconv/issues/1159

And if were to make them work they will be duplicating a test that is run in two other places. 

So I changed the name to what the test was actually doing it but I would be fine with removing them entirely.

Reply:
Shouldn't this test be checking that run_conversion runs properly when you pass a custom backend_configuration? How is this redundant with the other tests? What am I missing here?

Reply:
Yes, you are right, it is not redundant especially if I change both converter and interface versions.

Thinking about this I do want to get rid of these tests completely but I will open another PR describing my reasoning as I don't want this to get in the way.

I will revert this to just eliminate the passing of the nwbfile which does not make sense.